### PR TITLE
Update Angular documentation

### PR DESCRIPTION
**Pull request by stevehouel**
Add Angular Cli documentation

### CODE REVIEW

**Comment by stevehouel** [comment]
Tell me if it's enough for you regarding Angular Cli documentation

File: pages/using_angular.md [hunk 1]
```diff
@@ -158,3 +158,104 @@ The shorthand methods `success`, `info`, `warning` and `error` will have a defau
...
+[Class](https://github.com/angular/angular-cli/wiki/generate-class)         | `ng g class my-new-class`
+[Guard](https://github.com/angular/angular-cli/wiki/generate-guard)         | `ng g guard my-new-guard`
+[Interface](https://github.com/angular/angular-cli/wiki/generate-interface) | `ng g interface my-new-interface`
+[Enum](https://github.com/angular/angular-cli/wiki/generate-enum)           | `ng g enum my-new-enum`
+[Module](https://github.com/angular/angular-cli/wiki/generate-module)       | `ng g module my-module`
+
+
+### Test
+
+For consuitency purpose on JHipster application, tests execution are only available through `yarn` command:
```
Comment (deepu105) [inline pages/using_angular.md]: Spelling consistency

File: pages/using_angular.md [hunk 2]
```diff
@@ -158,3 +158,104 @@ The shorthand methods `success`, `info`, `warning` and `error` will have a defau
...
+
+```bash
+yarn test
+```
+
+### i18n
+
+JHipster is using `ng2-translate` dependency for translation purpose. Angular Cli i18n is based on default Angular tool.
+
+That's why, JHipster is not compatible with Angular Cli for now.
```
Comment (deepu105) [inline pages/using_angular.md]: With Angular CLI for i18n

File: pages/using_angular.md [hunk 3]
```diff
@@ -158,3 +158,104 @@ The shorthand methods `success`, `info`, `warning` and `error` will have a defau
             this.alerts
         )
     );
+
+## Using Angular Cli
+
+<div class="alert alert-info"><i>Info: </i>
+
+The Angular-Cli and JHipster can be used in parallel for development purpose and each of this options have their own configuration and build strategy. By default, JHipster is using his own when deploying application or for CI-CD integration.
```
Comment (deepu105) [inline pages/using_angular.md]: Its own

Comment (deepu105) [inline pages/using_angular.md]: Reword to Angular CLI everywhere for consistency

File: pages/using_angular.md [hunk 4]
```diff
@@ -158,3 +158,104 @@ The shorthand methods `success`, `info`, `warning` and `error` will have a defau
...
+
+</div>
+
+### Overview
+
+The Angular CLI is a tool to develop, scaffold and maintain Angular applications. JHipster integrate [Angular Cli](https://cli.angular.io/) tool for that purpose.
```
Comment (deepu105) [inline pages/using_angular.md]: Integrates

File: pages/using_angular.md [hunk 5]
```diff
@@ -158,3 +158,104 @@ The shorthand methods `success`, `info`, `warning` and `error` will have a defau
...
+
+This integration is done by adding a `.angular-cli.json` file on application root folder and his dependence on `package.json` file.
```
Comment (deepu105) [inline pages/using_angular.md]: File in the application root folder and its dependencies in the package.json

File: pages/using_angular.md [hunk 6]
```diff
@@ -158,3 +158,104 @@ The shorthand methods `success`, `info`, `warning` and `error` will have a defau
...
+
+For consuitency purpose on JHipster application, tests execution are only available through `yarn` command:
+
+```bash
+yarn test
+```
+
+### i18n
+
+JHipster is using `ng2-translate` dependency for translation purpose. Angular Cli i18n is based on default Angular tool.
```
Comment (deepu105) [inline pages/using_angular.md]: Is based on default Angular i18n support which is incompatible with Jhipster

**Comment by stevehouel** [comment]
I do it ASAP. Thx @deepu105 for those wording fix :)

**Pull request merged**